tests/test_state/test_load_state.py

```diff
@@ -0,0 +1,51 @@
+# pylint: disable=protected-access, arguments-differ
+from math import isclose
+
+import pytest
+from pendulum import now
+
+from gsy_e.models.state import LoadState
+from tests.test_state.test_consumption_state import TestConsumptionState
+
+
+class TestLoadState(TestConsumptionState):
+
+    @staticmethod
+    def _setup_base_configuration():
+        return LoadState(), now()
+
+    def _setup_configuration_for_settlement_posting(
+            self, energy_deviation=None, unsettled_deviation=None):
+        load_state, current_time_slot = self._setup_base_configuration()
+        if energy_deviation:
+            load_state._forecast_measurement_deviation_kWh[
+                current_time_slot] = energy_deviation
+        if unsettled_deviation:
+            load_state._unsettled_deviation_kWh[
+                current_time_slot] = unsettled_deviation
+        return load_state, current_time_slot
+
+    @pytest.mark.parametrize(
+        "desired_energy, traded_energy, measured_energy_consumption, expected_deviation",
+        [(2000.0, 1000.0, 1.8, 0.8), (2000.0, 2000.0, 1.8, -0.2), (2000.0, 2000.0, 2.0, 0.0)])
+    def test_set_energy_measurement_kWh_unsettled_energy_calculation(
+            self, desired_energy, traded_energy, measured_energy_consumption, expected_deviation):
+        """Test different case scenarios for unsettled energy:
+            1st overproduction,
+            2nd underproduction,
+            3rd no unsettled energy.
```

Comment:
Probably copy paste issue: `production` does not make sense for the load.

Reply:
Thanks! Is now fixed.